jobserver/templates/workspace_output_list.html

```diff
@@ -30,6 +32,47 @@
   </ol>
 </nav>
 
-{% include "outputs-spa.html" %}
+<div class="row">
+  <div class="col-lg-8 offset-lg-2">
+    <h3>Outputs</h3>
+
+    <p>
+      Listed below are the outputs for the {{ workspace.name }} workspace.
+    </p>
+
+    <ul class="list-group list-unstyled mb-4">
+      {% if user_can_view_all_files %}
+      <li class="list-group-item d-flex justify-content-between">
+        <div>
+          <a href="{{ workspace.get_current_outputs_url }}">Latest Files</a>
+        </div>
+        <div>
+          <span>{{ workspace.files.count }} file{{ workspace.files.count|pluralize }}</span>
+        </div>
+      </li>
+      {% endif %}
+
+      {% for snapshot in snapshots %}
+      <li class="list-group-item d-flex justify-content-between">
+
+        <div>
+          <a class="mr-3" href="{{ snapshot.get_absolute_url }}">
+            Outputs as of {{ snapshot.created_at|naturaltime }}
```

Comment:
This doesn't quite read right to me. I think a date would be better than natural time, and we only need to the day I think, minutes don't matter IMO.

e.g.

View Workspace Outputs:
  * Latest (or Current?)
  * Published on 2021-07-16 by bloodearnest
  * Published on 2021-06-15 by ghickman

Reply:
I'm not sure I like putting Latest in the same list as the snapshots. What about:
___________________
| View Current Outputs |
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
Draft Output Publications:
* 2021-07-16 by bloodearnest (2 files)

<!-- above is privileged only, below is everyone -->

Published Outputs:
* Published on  2021-07-16 by bloodearnest (5 files)
* Published on 2021-06-15 by ghickman (2 files)

Reply:
I really hate writing the text for these parts, so being given better words is hugely appreciated here!